models/articles.model.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection");
+
+
+async function selectArticleById(id) {
+    try {
+        const article = await db.query(`
+        SELECT articles.author, articles.title, articles.article_id, articles.body, articles.topic, articles.created_at, SUM(comments.votes)::INTEGER AS votes, articles.article_img_url FROM articles JOIN comments ON articles.article_id = comments.article_id WHERE articles.article_id=$1
```

Comment:
Is this accurately adding up to the total number of comments associated with each article in your test data? Or is it adding all of the comment votes together? Just double check - all cool if it is what you expect, if not though you may need to look up an alternative to SUM and look at what you are adding together (hint: check out COUNT in the postgresql docs)